mlrun/datastore/utils.py

```diff
@@ -180,3 +181,48 @@ def get_kafka_brokers_from_dict(options: dict, pop=False) -> typing.Optional[str
             FutureWarning,
         )
     return kafka_bootstrap_servers
+
+
+def transform_list_filters_to_tuple(additional_filters):
+    tuple_filters = []
+    if not additional_filters:
+        return tuple_filters
+    validate_additional_filters(additional_filters)
+    for additional_filter in additional_filters:
+        tuple_filters.append(tuple(additional_filter))
+    return tuple_filters
+
+
+def validate_additional_filters(additional_filters):
+    nan_error_message = "using NaN in additional_filters is not supported"
+    if (
+        additional_filters is None
+        or additional_filters == ()
+        or additional_filters == []
```

Comment:
How about the shorter
```suggestion
        additional_filters in [None, [], ()]
```
?